components/registry-facade/pkg/registry/manifest.go

```diff
@@ -185,13 +186,23 @@ func (mh *manifestHandler) getManifest(w http.ResponseWriter, r *http.Request) {
 				return err
 			}
 
+			originImageSize := 0
+			for _, layer := range manifest.Layers {
+				originImageSize += int(layer.Size)
```

Comment:
Oh, so we have the layer size available! :heart_eyes: 
Allthough we would need to map them to something meaningful to make sense of it...